perllib/FixMyStreet/App/Controller/Reports.pm

```diff
@@ -157,6 +157,8 @@ sub ward : Path : Args(2) {
             $c->detach;
         }
         $c->cobrand->call_hook('council_dashboard_hook');
+        $c->forward( 'load_contacts' );
```

Comment:
If this isn't .com (which is mandating login in the hook), anyone can then view these details at this page. I think this should only be a .com feature at present, so these two lines could be moved to the council_dashboard_hook and I think that would do it?